### PR TITLE
Add SIG CLI Annual Report 2022

### LINKED ISSUE #7102: 2022 Annual Report: SIG CLI
Chairs: @KnVerey @eddiezane @seans3 
Liaison: @justaugustus

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

A template has already been generated for your group, and is available [here](https://git.k8s.io/community/sig-cli/annual-report-2022.md).

Key dates:
- Initial PR to communtiy repo should be opened by March 24th, 2023
- PR should be reviewed and merged by April 7th, 2023

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/sig-cli/annual-report-2022.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig cli
/committee steering
/area annual-reports

### PR DESCRIPTION
**Pull request by eddiezane**
**Which issue(s) this PR fixes**:

Fixes #7102

### CODE REVIEW

File: sig-cli/annual-report-2022.md [hunk 1]
```diff
@@ -4,79 +4,87 @@
...
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Primary slack channel member count: 2361
+- Primary mailing list member count: 425
+- Primary meeting attendee count (estimated, if needed): 10+
+- Primary meeting participant count (estimated, if needed): 6
+- Unique reviewers for SIG-owned packages: 32
```
Comment (eddiezane) [inline sig-cli/annual-report-2022.md:81]: @soltysh you generated these numbers with https://github.com/MadhavJivrajani/owners-counts but they are suspiciously higher than what we reported last year.

https://github.com/kubernetes/community/blob/a055fba9ddab59a10db8b4149c9bea8d5a3a596b/sig-cli/annual-report-2021.md?plain=1#L81-L82

Comment (soltysh) [reply sig-cli/annual-report-2022.md:81]: Fair point, I haven't checked the output of the tool, not didn't compare that with last year. Probably worth double checking and reporting to @MadhavJivrajani a problem. I'm pretty sure we have more approvers, but you're right the delta is too big.

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**